core/src/main/scala/org/http4s/Uri.scala

```diff
@@ -1169,7 +1069,10 @@ object Uri {
             Ipv6Address(a, b, c, d, e, f, g, h)
         }
 
-      def repN[A](n: Int, p: cats.parse.Parser0[A], sep: cats.parse.Parser0[Unit] = P.unit): cats.parse.Parser0[List[A]] =
+      def repN[A](
```

Comment:
There's now a `repExactlyAs` in cats-parse that I think could replace this with concision and efficiency, but it doesn't have to be in this PR.